provider/postgis/postgis.go

```diff
@@ -453,7 +456,7 @@ func CreateProvider(config dict.Dicter, providerType string) (*Provider, error)
 
 	dbconfig, err := BuildDBConfig(uri.String())
 	if err != nil {
-		return nil, fmt.Errorf("Failed while building db config: %w", err)
+		return nil, fmt.Errorf("failed while building db config: %v", err)
```

Comment:
I'd keep `%w` here. (I guess this was changed by the original author?)
```suggestion
		return nil, fmt.Errorf("failed while building db config: %w", err)
```